src/browser/app/components/Input.js

```diff
@@ -0,0 +1,36 @@
+/* @flow */
+import type { Styled, InputTypes } from '../themes/types';
+import React from "react";
+import styled from "./styled";
+import Text, {TextProps} from "./Text";
+import Box from "./Box";
+
+export type InputProps = TextProps & {
+  name?: string,
+  label?: string,
+  labelSize?: string,
+  placeholder?: string,
+  maxLength?: number,
+  type?: InputTypes,
+  invalid?: boolean
+};
+
+const CustomInput = styled((theme, props: InputProps) => ({
+  $extends: Text,
+  display: 'block',
+  width: '100%',
```

Comment:
No better reason that I could not figure out how to create a styled component which might be a composition of components like the Input case with the label.